packages/perseus/src/perseus-api.tsx

```diff
@@ -100,6 +100,11 @@ export const ApiOptions = {
         // Indicates whether or not to use mobile styling.
         isMobile: PropTypes.bool,
 
+        // Whether to use v1 (Legacy) Keypad for MathInput mobile or
+        // use the new v2 Keypad
+        // TODO(matthewc)[LC-1088]: remove after v2 release
```

Comment:
**thought**: 🤔 I'm tempted to say we don't add your name here.  I think in webapp the standard is to just put the Jira ticket and ownership of that task is defined in Jira.